.chloggen/mx-psi_refactor-service-telemetry.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: all
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove go 1.19 support, bump minimum to go 1.20 and add testing for 1.21
+
+# One or more tracking issues or pull requests related to the change
+issues: [8207]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
Let's adopt the new changelog format and mark this as an API breaking change. Otherwise, it'll go to the changelog for collector end-users